pybloqs/static/__init__.py

```diff
@@ -71,7 +69,7 @@ def write(self, parent=None):
             if self._encode:
                 self.write_compressed(stream, f.read())
             else:
-                stream.write(f.read())
+                stream.write(f.read().decode())
```

Comment:
`.decode()` will default to the `ascii` codec on Python 2 so this will fail for UTF-8 encoded JS files.

You can either just `.decode('utf-8')` or you could use `io.open` which supports and encoding:
```python
from io import open
...
with open(self._local_path, encoding='utf-8') as f:
    stream.write(f.read().decode())
```